target/linux/ath79/dts/qca9563_dlink_dir-869-a1.dts

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "D-Link DIR-869 A1";
+	compatible = "dlink,dir-869-a1", "qca,qca9563";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+
+		led_power: power {
+			label = "dir-869-a1:orange:power";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		internet {
+			label = "dir-869-a1:white:internet";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Just looked at the manuel:
https://media.dlink.eu/ftp/products/dir/dir-869/documentation/DIR-869_man_revA_V1-02_eu_multi_20170516.pdf

So, this actually appears to be a power LED with two colors, so please use it like that. Based on the manual (as far as possible):

```
	aliases {
		led-boot = &led_power_orange;
		led-failsafe = &led_power_orange;
		led-running = &led_power_white;
		led-upgrade = &led_power_white;
	};

	leds {
		compatible = "gpio-leds";

		led_power_orange: power_orange {
			label = "dir-869-a1:orange:power";
			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
		};

		led_power_white: power_white {
			label = "dir-869-a1:white:power";
			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
		};
```